src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to make new metrics available in the Observability Platform in self-service.
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-metrics
+weight: 50
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is a ServiceMonitor?
+  - What is a PodMonitor?
+  - How do I ingest new metrics?
+  - Why do my clusters run the Prometheus Operator?
+---
+
+By default, all Giant Swarm clusters are equiped with the [Prometheus Operator](https://prometheus-operator.dev/) and a set of multiple [Prometheus](https://prometheus.io/) shards in `agent mode` functioning as monitoring agents to be able to collect and forward cluster critical and workload metrics to a central [Grafana Mimir](https://grafana.com/oss/mimir/) instance running on the management cluster.
+
+No workload is the same, especially in the way it exposes its metrics, so the Observability Platforms monitoring configuration needs to be flexible. That's why it is based on the `ServiceMonitor` and `PodMonitor` Custom Resource Definitions (CRDs) provided by the Prometheus Operator. 
+
+Those allow you to:
+
+- Define where the metrics you want to ingest are (for example the container or port)
+- Transform metrics before ingesting them (for example dropping unneeded data, adding extra labels)
+
+You can learn more about the `ServiceMonitor` and `PodMonitor` CRDs by checking the [Prometheus Operator API Docs](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md).
+
+## Prerequisites
+
+Before you start to ingest data from a container running in your clusters, you need to make sure that your application is already [instrumented](https://opentelemetry.io/docs/concepts/instrumentation/) to export metrics and make sure the metrics it provides are useful to you.
```

Comment:
```suggestion
Before you start to ingest data from a container running in one of your clusters, you need to make sure that your application is already [instrumented](https://opentelemetry.io/docs/concepts/instrumentation/) to export metrics and make sure the metrics it provides are useful to you.
```

Reply:
Does that change anything to the message?

Reply:
From my understanding, in your original sentence you imply that one is going to ingest data from a container running simultaneously in several clusters, which is well... Impossible :sweat_smile: 

But maybe I am the weird one here

Reply:
Maybe we can just replace it with:
```suggestion
Before you start to ingest data from a running container, you need to make sure that your application is already [instrumented](https://opentelemetry.io/docs/concepts/instrumentation/) to export metrics and make sure the metrics it provides are useful to you.
```